apps/aeutil/lib/serialization.ex

```diff
@@ -57,4 +57,19 @@ defmodule Aeutil.Serialization do
       merkle_proof(tail, acc)
     end
   end
+
+  @spec pack_binary(term()) :: map()
+  def pack_binary(term) do
+    case term do
+      %Block{} ->
+        Map.from_struct(%{term | header: Map.from_struct(term.header)})
+      %SignedTx{} ->
+        Map.from_struct(%{term | data: Map.from_struct(term.data)})
+      %{__struct__: _} ->
+        Map.from_struct(term)
+      _ ->
+        term
+    end
+    |> Msgpax.pack!(iodata: false)
+  end
```

Comment:
Can we get a deserialization function? It might be useful in future.

Reply:
I think this is hard to do right since you can pack almost anything, if we need it in the future we can do that depending on what we need to deserialize.